requirements/base.txt

```diff
@@ -204,11 +204,11 @@ web-fragments==2.2.0
     # via xblock
 webob==1.8.7
     # via xblock
-xblock==4.0.1
+xblock==5.0.0
```

Comment:
Hmm.. I think this update should wait until edx-platform is using `xblock==5.x`.